lib/app.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    puts 'Welcome to School Library App!'
+  end
+
+  def list_books
+    puts
+    if @books.empty?
+      puts 'No books found.'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
+    puts
+    if @people.empty?
+      puts 'No people found.'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def list_rentals
+    puts
+    puts 'ID of person:'
+    person_id = gets.chomp.to_i
+    rentals = @rentals.select { |rental| rental.person_id == person_id }
```

Comment:
- [ ] I'm getting this error here:

![image](https://github.com/hajnaloltyan/school-library/assets/77069011/276a248b-89c0-499b-a485-16391adea21f)

Please check the `person_id` attribute that doesn't exist! 🤓